activestorage/test/template/image_tag_test.rb

```diff
@@ -11,22 +11,26 @@ class ActiveStorage::ImageTagTest < ActionView::TestCase
   end
 
   test "blob" do
+    travel_to Time.now
```

Comment:
Are these `travel_to` calls now necessary for these tests to pass?

Reply:
Yes

Reply:
Ah, I see, you need to freeze time because otherwise `image_tag` and `polymorphic_url` would now produce different URLs.  What do you think about calling `freeze_time` in `setup` and adding a comment to explain its purpose?

Also, could this impact system / integration tests for apps already using Active Storage?

Reply:
Yes, `freeze_time` would work (I actually had that in my local copy, not pushed yet).

As for the second question, functionally the behavior remains the same during 5 minutes, so tests should run fine in general AFAIK.